test/services/twilio-help/twilio-help.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const { expect, test } = require('@oclif/test');
+const { Command, flags } = require('@oclif/command');
+const stripAnsi = require('strip-ansi');
+
+const TwilioHelp = require('../../../src/services/twilio-help/custom-help');
+
+class DummyCommand extends Command {
```

Comment:
If possible Can we add the test case for one existing command also(like `twilio profiles:remove`)?

Reply:
This feels to me more of an integ test case. 
I see UTs to test overall behaviour of a class, without dependence on other software parts. In case commands like `profiles:remove` change in future as well, we'd be requiring changes here as well (without changes in help functionality!), and seems undesired. 

Added a DummyCommand in this class to assert this behaviour for the same reason, so we can decouple dependence on a specific command.